addons/TBMod_arsenal/arsenal/fn_arsenalVANILLA.sqf

```diff
@@ -564,7 +562,6 @@ _items append (switch (ACE_player getVariable ["TB_rolle", ""]) do
             // Items
             "MineDetector",
             "ACE_wirecutter",
-            "ACE_Clacker",
```

Comment:
Warum der raus? Ist auch in allen anderen drin.

Reply:
Bei einer Eron Mission wurde der kleine aus unwissenheit mitgenommen und so konnte nicht simultan gesprengt werden.... es sind halt 5 km vs 250 m und ob ich jetzt nen halben kilo mehr wiege oder nicht kommt jetzt nicht drauf an :D

Reply:
Ich persönlich bin gegen das Entfernen... es gilt aufklären nicht zuschneiden bei sowas aus meiner Sicht, aber gerne andere eigene Meinungen dazu...
@TacticalBaconDevs/developer @TacticalBaconDevs/helper

Reply:
Ich wäre für eine Beibehaltung beider Zünder, um einem Missionsbauer die Freiheit zu lassen auf den 250er zu beschränken und von den Jungs taktisches Verständnis zu erwarten.